Editor/RaymarchShader.cs

```diff
@@ -275,8 +275,11 @@ struct v2f {
                     half3 directionGrid = directionWorld;
                     half3 invDirectionGrid = 1.0 / directionGrid;
 
-                    float iBlockGridBlocks = floor((gridSize + blockSize - 1) / blockSize);
-                    float3 blockGridSize = iBlockGridBlocks * blockSize;
+                    int3 iGridSize = int3(round(gridSize));
+                    int  iBlockSize = int(round(blockSize));
+                    int3 iBlockGridBlocks = (iGridSize + iBlockSize - 1) / iBlockSize;
+                    int3 iBlockGridSize = iBlockGridBlocks * iBlockSize;
+                    half3 blockGridSize = half3(iBlockGridSize);
```

Comment:
```suggestion
                    float3 iBlockGridBlocks = floor((gridSize + blockSize - 1) / blockSize);
                    float3 blockGridSize = iBlockGridBlocks * blockSize;
```

The issue I've introduced was that iBlockGridBlocks should be a vector, not a scalar. The suggestion should bring back the correct behavior while still using float math, therefore not throwing warnings in Unity.